src/openapi.yaml

```diff
@@ -7295,8 +7352,6 @@ components:
           format: uri
           description: Form to which the JSON logic applies.
         jsonLogicTrigger:
-          type: object
-          additionalProperties: {}
```

Comment:
This one is actually correct, since you can provide a boolean like `true` to always trigger.